packages/controls-navigator/src/NavigatorControl.vue

```diff
@@ -0,0 +1,293 @@
+          :editable="navigationType === 'tab' || currentStep > index"
+          :title="prompts[index - 1] ? prompts[index - 1].name : ''"
+          :complete="getComplete(index)"
+          :class="getStepClass(currentStep, index)"
+          @click="gotoStep(index)"
+        />
+        <div class="breadcrumbs-container">
+          <YOUIBreadcrumbs
+            :id="`breadcrumbs-${index}`"
+            class="breadcrumbs"
+            :breadcrumbs="answers[prompts[index - 1] ? prompts[index - 1].name : undefined]"
+          />
+        </div>
+      </template>
+    </v-stepper>
+  </div>
+</template>
+
+<script setup>
+import { computed, ref, toRefs, unref, watchEffect } from "vue";
+import YOUIBreadcrumbs from "./YOUIBreadcrumbs.vue";
+import * as _ from "lodash";
+const props = defineProps({
+  promptIndex: {
+    type: Number,
+    default: 0,
+  },
+  prompts: {
+    type: Array,
+    default: () => [],
+  },
+  promptAnswers: {
+    type: Object,
+    default: () => {},
+  },
+  allAnswers: {
+    type: Object,
+    default: () => {},
+  },
+  navigationType: {
+    type: String,
+    default: "tab",
+  },
+});
+
+const { promptIndex, prompts, allAnswers, promptAnswers, navigationType } = toRefs(props);
+
+const emits = defineEmits(["onGotoStep"]);
+
+const answers = ref({});
+
+const currentStep = ref(0);
+
+const travel = ref({});
+
+const steps = computed(() => {
+  return unref(prompts).length;
+});
+
+watchEffect(() => {
+  currentStep.value = unref(promptIndex) + 1;
+});
+
+watchEffect(() => {
+  const count = unref(prompts).length;
+  for (let i = 0; i < count; i++) {
+    travel.value[i + 1] = false;
+  }
+});
+
+watchEffect(() => {
+  if (unref(promptAnswers) && unref(promptAnswers).promptName && unref(promptAnswers).answers) {
+    // Vue 2 requires a new object for reactivity to trigger updates
+    answers.value = Object.assign({}, unref(answers), {
+      [unref(promptAnswers).promptName]: unref(promptAnswers).answers,
+    });
+  }
+});
+watchEffect(() => {
+  if (unref(allAnswers) && !_.isEmpty(unref(allAnswers))) {
+    // Vue 2 requires a new object for reactivity to trigger updates
+    answers.value = Object.assign({}, unref(answers), unref(allAnswers));
+  }
+});
+const getStepClass = (currentStep, index) => {
+  return {
+    "step-linkable": navigationType.value === "tab" ? true : currentStep > index,
+  };
+};
+const gotoStep = (index) => {
+  if (navigationType.value === "tab") {
+    currentStep.value = index;
+    emits("onGotoStep", index - 1);
+  } else {
+    // numOfSteps is number of steps to go back
+    const numOfSteps = currentStep.value - index;
+    if (numOfSteps > 0) {
+      emits("onGotoStep", numOfSteps);
+    }
+  }
+};
+const isTraveled = (index) => {
+  travel.value[currentStep.value] = true;
+  return index !== currentStep.value && !!travel.value[index];
+};
+const getNavigateClass = () => {
+  return navigationType.value === "tab" ? "PTNavigationTabClass" : "PTNavigationStepperClass";
```

Comment:
Please remove all references to `PT` so for example here it should be something like  `return navigationType.value === "tab" ? "NavigatorControlTabClass" : "NavigatorControlStepperClass";`